### ISSUE TITLE
Support Tiled's Isometric Staggered

### ISSUE THREAD

**Issue by jtmcdole** [flame-engine/flame#1893]
# What could be improved
Support the last layout featured in Tiled - Isometric Staggered.  In this mode, the tiles do not follow the isometric coordinate system, but instead resemble the hexagonal staggering.

# Why should this be improved
I'm a completionist.

# Any risks?
None.

# More information
I'll work on this PR

**Comment by jtmcdole** [comment]
Slightly on hold as the current use of `SpriteBatch` on RenderableTileMap is breaking in all contexts where there could be tile overlaps (staggering, tall tiles, etc).

**Comment by spydon** [comment]
> Slightly on hold as the current use of `SpriteBatch` on RenderableTileMap is breaking in all contexts where there could be tile overlaps (staggering, tall tiles, etc).

Any idea how that could be solved?

**Comment by jtmcdole** [comment]
> Any idea how that could be solved?

I have a bug open for it and ideas that are planning out!  This is unblocked.
